src/centos/scripts/start-marklogic.sh

```diff
@@ -429,6 +429,8 @@ fi
 ################################################################
 if [[ -f /var/opt/MarkLogic/DOCKER_JOIN_CLUSTER ]]; then
     info "MARKLOGIC_JOIN_CLUSTER is true, but skipping join because this instance has already joined a cluster."
+elif [[ "${MARKLOGIC_CLUSTER_TYPE}" == "bootstrap" ]] && [[ $(hostname) == *-0 ]]; then
```

Comment:
This will not work on docker as the hostname is not the same as Kubernetes.

Reply:
If only runs in docker, it will be fine. As this bug happens when MARKLOGIC_JOIN_CLUSTER flag sets to true. In Docker, the MARKLOGIC_JOIN_CLUSTER will be set to false.

This is bug happens in Kubernetes because Kubernetes always set the MARKLOGIC_JOIN_CLUSTER to true. When it first starts, there will be no issue. But when it restarts, it will still calls this line: BOOTSTRAP_STATUS=$(verify_bootstrap_status "${MARKLOGIC_BOOTSTRAP_HOST}"),
verify_bootstrap_status will call get_host_id function where there will be a curl call with --cacert. For Kubernetes Bootstrap cluster, the cacert is empty because we get the cacert in init-container. All other pods will have the cacert except for the boostrap host. 

So the solution I provide here is to add another check to see if it is bootstrap cluster and bootstrap host. This way it could call the function that fails the whole setup.